src/ui/pages/GameUI.jsx

```diff
@@ -54,22 +56,32 @@ const GameUI = () => {
     }
   }, [gameEngine, uiProxy]);
 
-  return (
-    <main css={wrapper}>
-      <PageHeader>Level 1</PageHeader>{' '}
-      <section css={gameBorder}>
-        <canvas css={{ width: '65vw' }} height={512} width={1024} id='GameCanvas' ref={ref} />
+  const levelExists = () => {
+    const levelsArray = Array(levels)
+      .fill()
+      .map((x, i) => i + 1);
+
+    return levelsArray.includes(Number(params.levelSelectId));
+  };
 
-        <div css={buttonWrapper}>
-          <Link to='/'>
-            <Button type='silver'>
-              change <br />
-              level
-            </Button>
-          </Link>
-        </div>
-      </section>
-    </main>
+  return (
+    <>
+      {!levelExists() ? <Navigate to='/level-select' replace /> : ''}
```

Comment:
Jak sobie zmieniłem warunek, to wtedy wyświetla mi się poziom 2 i 3, natomiast 1 się nie wyświetla. Więc moim zdaniem coś jest nie tak z tą funkcją, która sprawdza czy poziom istnieje. (Albo źle te poziomy tworzę i nie potrzebnie zawracam głowę)

Reply:
No tak, bo wtedy odwróciłeś warunek, więc będą Ci wchodzić 2, 3 ale też wszystkie numery(a tego nie chcemy) poza 1 (bo ten level teoretycznie istnieje, na podstawie tej zmiennej o której pisałam w komentarzu).